rust_icu_unum/src/lib.rs

```diff
@@ -263,12 +271,83 @@ impl UNumberFormat {
         format_double_currency_impl(
             versioned_function!(unum_formatDoubleCurrency),
             self.rep.as_ptr(),
-            number, 
+            number,
             currencyz.as_mut_c_ptr(),
             0 as *mut sys::UFieldPosition,
         )
     }
-   
+
+    /// Implements `unum_parseToUFormattable`. Since 0.3.1.
+    ///
+    /// > **WARNING** the `parse_position` parameter is with respect to the number index
```

Comment:
"with respect to the number index" is a bit unclear. Maybe something like the "starting index of the numeric string"?